### PR TITLE
16: Remove sass question and libsass/ruby sass options

### PR DESCRIPTION
**Pull request by mortona42**
Removed questions from prompting.   Remove ruby sass grunt config.  Skip check for libsass option - just config grunt for it if grunt is selected.

Work related to #16

### CODE REVIEW

File: web-starter/index.js [hunk 1]
```diff
@@ -129,71 +102,48 @@ module.exports = generators.Base.extend({
       }
     },
     gruntLibSass : function() {
-      if (this.config.get('sass') === SASS_CHOICES[0]) { //lib sass
```
Comment (mortona42) [inline web-starter/index.js:132]: I just removed this check - run this config if using grunt (should be always).

File: web-starter/index.js [hunk 2]
```diff
@@ -164,11 +164,7 @@ module.exports = generators.Base.extend({
           case 'drupal':
             promise = drupal_modules.getLatestMinorVersions('gesso')
               .then(function(versions) {
-                // If the user selected Libsass use the most recent release
-                // Otherwise use the most recent release of 7.x-1.x
-                var url = (SASS_CHOICES[0] === that.config.get('sass')) ?
-                    _.find(versions, { version_major : 7 }).download_link :
-                    _.find(versions, { version_major : 7, version_minor : 1 }).download_link;
+                var url = _.find(versions, { version_major : 7 }).download_link;
```
Comment (mortona42) [inline web-starter/index.js:166]: Is this the right version for drupal 7 + libsass?

Comment (coreylafferty) [reply web-starter/index.js:166]: If that'd download the latest D7 release (currently 7.x-2.0), then yes.

**Comment by coreylafferty** [comment]
@mortona42 I know we've discussed reading the Grunt tasks from the theme itself, but if you're updating the currently installed root Grunt tasks to get them working until that's complete, the latest versions are what you see in the /tasks/ folder and any relevant root files (e.g., package.json, bower.json, etc.) here:

D8: https://github.com/forumone/gesso
WP: https://github.com/forumone/gesso-wp/ (identical to the D8 tasks)
D7: https://github.com/forumone/gesso/tree/7.x-2.x

I believe the only difference between these files and what would end up in the project root would be 1) the theme path in package.json and 2) any additional modules or tasks that web-starter itself needs.

**Pull request merged**